luarules/gadgets/dev_replay_data.lua

```diff
@@ -10,12 +10,15 @@ function gadget:GetInfo()
 	}
 end
 
+--Spring
+local spIsReplay = Spring.IsReplay

```

Comment:
You could probably call the function here, once, and then just use the bool.

Reply:
Done, and tested on my end to still work with just using the bool